# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/painting/RetainedDisplayListBuilder.h
# Commit: d5855a08e601
# Full Hash: d5855a08e6016ad30e8e0a0ef88149736d5f2675
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2024-03-19 09:35:23
# Regressor Bug: 1860328
# File Overlap Count: 2
# Description:
#   Bug 1860328 - Fix a bug where caret misses when dragging and dropping r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192377
# ==============================================================================

diff -r 7e490d41e79e -r d5855a08e601 layout/painting/RetainedDisplayListBuilder.h
--- a/layout/painting/RetainedDisplayListBuilder.h	Mon Mar 18 08:49:10 2024 +0000
+++ b/layout/painting/RetainedDisplayListBuilder.h	Mon Mar 18 09:12:57 2024 +0000
@@ -272,7 +272,6 @@
 
   nsDisplayListBuilder mBuilder;
   RetainedDisplayList mList;
-  WeakFrame mPreviousCaret;
   RetainedDisplayListMetrics mMetrics;
   RetainedDisplayListData mData;
 };